te/te_docker/setup_postgres.sh

```diff
@@ -32,7 +32,7 @@
 apt-get purge -y postgresql
 apt-get install -y wget
 wget --no-check-certificate --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main" > /etc/apt/sources.list.d/PostgreSQL.list'
+sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ jammy-pgdg main" > /etc/apt/sources.list.d/PostgreSQL.list'
```

Comment:
We might not need to get postgres from custom postgres repo. We could get it from ubuntu repo. Could we please consider removing this ?

Reply:
Also move postgres to the latest version available. IG Ubuntu support psql 14